lib/proxy.rb

```diff
@@ -111,7 +111,7 @@ def add_authenticated_user_organisation_header(env)
   end
 
   def healthcheck_path?(path)
-    path == "/healthcheck"
+    %w[/healthcheck /healthcheck/live /healthcheck/ready].include? path
```

Comment:
Could you expand the commit message a little?

Reply:
Done